ui_testing/pages/orders_page.py

```diff
@@ -105,3 +111,22 @@ def order_filters_element(self, key='all'):
             return filter_fileds
         else:
             return filter_fileds[key]
+
+    def get_random_main_order_with_sub_orders_data(self):
+        self.info('+ Get Main order data with related subOrders')
+        # get all the order rows
+        all_orders = self.base_selenium.get_table_rows(element='orders:orders_table')
+        # select random order
+        row_id = randint(0, len(all_orders) - 2)
```

Comment:
you can use get_random_table_row it does the same purpose

Reply:
@mazenmmbadr but i'm using the `row_id` later in this function to get the suborder data of this random row, `get_random_table_row` doesn't return the `row_id`